python/main.py

```diff
@@ -85,4 +98,24 @@ def get_itemsid(item_id:int):
     except IndexError:
         raise HTTPException(
             status_code=404, detail=f"item_id {item_id} not exist"
-        )
\ No newline at end of file
+        )
+@app.get("/search")
+def get_keyword(keyword: str):
+    conn = sqlite3.connect(dbpath)
+    cursor = conn.cursor()
+    cursor.execute("""SELECT items.name, category.name 
+                    FROM items
+                    INNER JOIN category ON items.id = category.id
+                    WHERE items.name like ?""", (keyword,))
```

Comment:
From [Step 4](https://github.com/mercari-build/mercari-build-training-2023/blob/main/document/04-database.en.md#2-search-for-an-item), it says `include` so maybe partial matched result should be returned.

> Make an endpoint to return a list of items that include a specified keyword called GET /search.

For example, keyword=jack also return name=jacket.

Also, if you want exact match, WHERE clause should be `=`, not `like`.